### PR TITLE
Cache the AP delivery process

### PR DESCRIPTION
**Pull request by annando**
This should speed up the delivery.

### CODE REVIEW

**Comment by MrPetovan** [comment]
Follow-up to #5794

File: src/Protocol/ActivityPub/Transmitter.php [hunk 1]
```diff
@@ -480,6 +481,27 @@ private static function getTypeOfItem($item)
...
+	/**
+	 * @brief Creates the activity or fetches it from the cache
+	 *
+	 * @param integer $item_id
+	 *
+	 * @return array with the activity
+	 */
+	public static function createCachedActivityFromItem($item_id)
+	{
+		$cachekey = "APDelivery:createActivity:".$item_id;
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub/Transmitter.php]: Code standards: Please wrap operators with spaces.

File: src/Protocol/ActivityPub/Transmitter.php [hunk 2]
```diff
@@ -480,6 +481,27 @@ private static function getTypeOfItem($item)
 		return $type;
 	}
 
+	/**
+	 * @brief Creates the activity or fetches it from the cache
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub/Transmitter.php]: If the function description is a one-liner, I don't think you need the non-standard `@brief` docblock variable. It stunts some documentation parsers that thinks there's no description at all. `@brief` should be use in addition to a regular description (without docblock variable in front of it).

Comment (annando) [reply src/Protocol/ActivityPub/Transmitter.php]: What exactly should I do now? I used the ```@brief``` oneliner for **every** function in the new files. So removing it just at this single place doesn't make much sense.

Comment (MrPetovan) [reply src/Protocol/ActivityPub/Transmitter.php]: Fair enough.

**Pull request merged**